### PR TITLE
Update My New File

### PR DESCRIPTION
**Pull request by Rowdy2018**
More instructions

### CODE REVIEW

File: My New File [hunk 1]
```diff
@@ -6,3 +6,8 @@ It is about checking to see if we can have a private repository
 Let's see what we can do
 
 Adding some more content here
+
```
Comment (Lal2023) [inline My New File:9]: Need to add another line here to say what we are going to do

**Review by Lal2023 — COMMENTED** [review]
Pls check my comments and amend the file

File: My New File (hunk 1 above)
Comment (Rowdy2018) [reply My New File:9]: Can you pls let me know what to add

Comment (Lal2023) [reply My New File:9]: Add the following line:

We are going to start with bla bla bla

**Review by Lal2023 — APPROVED** [review]
Sounds good

**Comment by Lal2023** [comment]
Successful

**Pull request merged**